crates/re_viewer/src/app.rs

```diff
@@ -1325,6 +1334,56 @@ impl App {
             1.0
         }
     }
+
+    #[allow(dead_code)] // only used in web
+    pub(crate) fn toggle_fullscreen(&self) {
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            let egui_ctx = &self.re_ui.egui_ctx;
+            let fullscreen = egui_ctx.input(|i| i.viewport().fullscreen.unwrap_or(false));
+            egui_ctx.send_viewport_cmd(egui::ViewportCommand::Fullscreen(!fullscreen));
+        }
+
+        #[cfg(target_arch = "wasm32")]
+        {
+            if let Some(options) = &self.startup_options.fullscreen_options {
+                // Tell JS to toggle fullscreen.
+                options.on_toggle.call().unwrap();
```

Comment:
Let's replace the unwraps with a log call using one of the helpers in `crates/re_log/src/result_extensions.rs`

Reply:
couldn't use those because `JsValue` does not implement `Display` - used the web tools `string_from_js_value` instead